### DISCUSSION TITLE
GLPI-AgentMonitor v1.1.0

### DISCUSSION THREAD

**Discussion by g-bougard** [glpi-project/glpi-agentmonitor#6] [Announcements]
I just released v1.1.0.

It includes few new features developed by @redddcyclone

@redddcyclone Thank you for your efforts.

I updated GLPI-Agent MSI packaging to include an option to install GLPI-AgentMonitor v1.1.0. @redddcyclone updated the monitor code so it can be cleanly managed during installations & upgrades. Thank you again ;-)

With next GLPI-Agent nightly release, you should be able to try this integration. For silent installation, you should set `AGENTMONITOR=1` on the msiexec command line.

**Comment by redddcyclone** [comment]
Hi @g-bougard 

That's great! I'm glad things worked out fine and the Monitor could be integrated in the MSI installer 😄

Thanks for all the help and your suggestions on improving the code! I will keep adding new features and fixing things as soon as I can 🚀

**Comment by redddcyclone** [comment]
@g-bougard 

Should the MSI graphical installer have an option to install the Monitor? The latest nightly build doesn't have it. Or should it only be installed when using `AGENTMONITOR=1` on the command line?

**Comment by g-bougard** [comment]
hI @redddcyclone 

there's an option. You will find it in the frame where you choose the running mode. I did this as we should only be able to install it if the service mode has been choosen.

**Comment by redddcyclone** [comment]
@g-bougard 

Ah, yes, I forgot to uncheck "Quick installation" 😅 I'm sorry, looks like everything is fine

**Comment by redddcyclone** [comment]
Hi @g-bougard 

We had a little problem. Yesterday I got an issue report about the "New ticket" button. It wasn't working with HTTP URLs, only HTTPS. I commited the fix. Could you release a new version before v1.1.0 gets released with the Agent?

Also, I added a service control button and an italian translation.

Thanks a lot

**Comment by g-bougard** [comment]
Hi @redddcyclone 

sure, I'll prepare a 1.1.1 asap.
